internal/engine/build_and_deployer.go

```diff
@@ -21,6 +21,7 @@ type BuildAndDeployer interface {
 
 	// PostProcessBuild gets any info about the build that we'll need for subsequent builds.
 	// In general, we'll store this info ON the BuildAndDeployer that needs it.
+	// Each implementation of PostProcessBuild is responsible for executing long-running steps async.
```

Comment:
did you intend to add async in this pr?

Reply:
it looks like you tried to add it in one commit, then reverted it in the second commit

Reply:
...oh, awkward, yes. i just failed to commit it oops.

Reply:
😳